nidirect_news/src/Controller/NewsListingController.php

```diff
@@ -67,22 +67,24 @@ public static function create(ContainerInterface $container) {
   public function default() {
     $content = [];
 
-    // Latest news embed display.
-    $content['latest_news'] = [
-      '#type' => 'view',
-      '#name' => 'news',
-      '#display_id' => 'latest_news',
-    ];
+    if (empty($this->requestStack->getCurrentRequest()->query->get('page'))) {
```

Comment:
If they click on a pager link, then back to page 1 it'll have the page parameter, I think you'll need an extra check in here for page=0

Reply:
Yes, I spotted that. It looks like when you have `page=0` the value is passed into `empty()` and it's the same outcome as when the parameter is missing 👍

Reply:
Ahh sweet 👍